color.go

```diff
@@ -2,10 +2,23 @@ package main
 
 import "strings"
 
-var colorPalette = []string{
+var colorPalette = [][]string{
+	{
+	`"#ffbb00"`, `"#f90085"`, `"#00bfb2"`, `"#008ffc"`, `"#fc4f00"`, `"#9d00ff"`, `"#ff0000"`, `"#00b515"`, 
+	`"#c7f400"`, `"#f9a2a2"`, `"#007072"`, `"#e6beff"`, `"#aa5b00"`, `"#fff266"`, `"#7f0000"`, `"#aaffc3"`,
+	`"#7f7f00"`, `"#ffe0c1"`, `"#000080"`, `"#808080"`, `"#000000"`},
```

Comment:
The colours look awesome but if I'm counting correctly the new palette only has 2 more colours, which unfortunately defeats the purpose of https://github.com/marianogappa/chart/issues/27

Let's just add more colours here.